pipelines/defaults.json

```diff
@@ -90,6 +90,16 @@
         "defaultDynamicParas": {
             "testLists"      : ["sanity.functional", "extended.functional", "sanity.system", "extended.system", "special.system", "sanity.jck", "extended.jck", "sanity.openjdk", "dev.system"],
             "numMachines"    : ["3", "3", "3", "3", "5", "4", "8", "2", "5"]
+        },
+        "ebcEnabledTargets":{
+            "linux_x64" :       [""],
```

Comment:
Might as well add sanity.functional here so we can turn it on with this PR.
Is the execution time on EBC similar to static nodes?
How much overhead to set them up?
If we request 3, are they setup in parallel?
We should probably increase the numMachines for the EBC enabled targets, however, those numbers aren't per platform, only per target, so increasing them would increase for all platforms. We may want to consider changing that code to be numMachines per platform per target.

Reply:
For adding sanity.functional refer to our discussion about test on all flavours of linux, it is not possible until all others become available. 
I will add this question to related tasks. But it has to be. We can also provision even more resources or nodes to be even faster. 
Overhead is ~15 min per node. 
Yes. 3 in parallel.
The change per platform is comes from the test design as I see. So I am not sure if we can address it in this change.

Reply:
I see your point. Let me do one of them and add it here. Might take longer than other your comments.

Reply:
@AdamBrousseau here is a final test:
https://hyc-runtimes-jenkins.swg-devops.com/job/build-scripts/job/jobs/job/jdk22/job/jdk22-linux-x64-openj9/110/console

In that sanity.perf test https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk22_j9_sanity.perf_x86-64_linux/24/parameters/ is done using EBC. 

We can enable sanity.perf inside that.